src/docs/environment/index.mdx

```diff
@@ -213,16 +234,17 @@ And after doing that, reload your shell:
 exec "$SHELL"
 ````
 
-Any time the `.envrc` configuration changes (including the first time) you will be prompted to run `direnv allow` before any
+Any time the `.envrc` configuration changes (including the first load) you will be prompted to run `direnv allow` before any
 of the configuration will run. This is for security purposes and you are encouraged to inspect the changes before running this command.
 
 ### Customize your development environment variables
 
 If you want to personalize your environment variables, you can do so by creating a `.env` file. This file is
-ignored by `git`, thus, you will not be able to leak it into one of your PRs.
+ignored by `git`, thus, you will not be able to leak it into one of your PRs. 
 
-Two examples of variables that you can place there are `SENTRY_SILENCE_DIRENV_NOTICE` to make `direnv` output less noisy
-and `SENTRY_DEVENV_NO_REPORT=1` to prevent the development environment to report errors to Sentry.
+One of `bootstrap`'s steps [creates a starter `.env`](https://github.com/getsentry/sentry/blob/master/.env.example) which should 
```

Comment:
This link points to a nonexistent destination until https://github.com/getsentry/sentry/pull/26530 is merged in.